src/settings-ui/Settings.UI/ViewModels/GeneralViewModel.cs

```diff
@@ -290,6 +301,18 @@ public bool AutoDownloadUpdates
             }
         }
 
+        public bool IsAutoDownloadUpdatesCardEnabled
+        {
+            get => !AutoUpdatesDisabledOnDevBuild && !_autoDownloadUpdatesIsGpoDisabled;
+        }
+
+        // The settings card is hidden for users who are not a member of the Administrators group and in this case the GPO info should be hidden too.
```

Comment:
note that this will be changed once per-user installer is out

Reply:
Okay. But then we need to update it after user install is out. I only improved code.

Currently it is hidden so gpo message should be too.

Reply:
yeah, yeah, it was note to myself mostly :)